sdk/src/Vyper.ts

```diff
@@ -87,5 +109,63 @@ export class Vyper {
 
         return trancheConfig;
     }
+
+    async getRedeemLendingConfiguration(redeemLendingStateId?: PublicKey) {
+
+        if (!redeemLendingStateId) {
+            redeemLendingStateId = this.redeemLendingStateId;
+        }
+        const redeemLendingState = await this.redeemLendingProgram.account.redeemLogicConfig.fetch(redeemLendingStateId);
+        const redeemLogicState = new RedeemLogicState(
+            redeemLendingState.interestSplit,
+            redeemLendingState.fixedFeePerTranche.toNumber(),
+            redeemLendingState.owner
+        )
+        return redeemLogicState;
+    }
+
+    async getRateMockState(rateMockStateId?: PublicKey) {
+
+        if (!rateMockStateId) {
+            rateMockStateId = this.rateMockStateId;
+        }
+        const rateMockState = await this.rateMockProgram.account.rateState.fetch(rateMockStateId);
+        const rateState = new RateState(
+            rateMockState.fairValue,
+            rateMockState.refreshedSlot.toNumber(),
+        )
+        return rateState;
+    }
+
+    async refreshTrancheFairValue(fairValue: number, trancheId?: PublicKey): Promise<string> {
```

Comment:
the fair value won't be available as a parameter in production, right now we can provide it manually to the rate-mock plugin because it's just a _mock_ plugin, but in production we'll have real plugins with just a refresh method to fetch the reserve fair value from the associated platform

Reply:
Created a method for instruction as well as rpc call and removed from here.